pkg/controller/migmigration/task.go

```diff
@@ -184,11 +202,12 @@ func (t *Task) Run() error {
 	t.Log.Info("[RUN]", "stage", t.stage(), "phase", t.Phase)
 
 	t.init()
+	step := t.setStep()
 
 	// Run the current phase.
 	switch t.Phase {
 	case Created, Started:
-		t.next()
+		t.next(step)
```

Comment:
Please explain how this change is related to and required for this feature.  Else, revert.